lib/transforms/search-results-to-jsonapi.js

```diff
@@ -27,7 +27,7 @@ module.exports = (queryParams, results, config) => {
 // Create resources from hits, filter any unknown resource types
 function createResources (results, rootUrl, mediaPath) {
   const hits = (results.hits || {}).hits || [];
-
+  console.log(hits.length);
```

Comment:
This is fine to _test_ the output but will be quite annoying later.
please make a note to remove it. and/or use "Good" for Logging. see: https://github.com/dwyl/learn-hapi#logging-with-good